_syllabus/seed.py

```diff
@@ -76,12 +77,19 @@
         hwDate  = lecDate.replace(hours=15)
         solDate = hwDate.shift(weeks=1)
 
+
+	 
+	oh_plus_plus_link = wk{0:02d}-ohpp.md'.format(idx + 1)
+
```

Comment:
I was thinking something like this:

#### syllabus.html
```html
{% if page.ohppRelease %}
  {% assign now = 'now' | date: '%s' | times: 1 %}
  {% assign ohppRelease = page.ohppRelease | date: '%s' | times: 1 %}
  {% if now > ohppRelease %}
    <p><a href="https://github.com/c4cs/c4cs.github.io/blob/master/static/w17/oh-plus-plus/wk{{ page.week }}-ohpp.md">OH++ Week {{ page.week }} [GitHub]</a></p>
  {% endif %}
{% endif %}
```

#### seed.py
```python
template = """---
week: {week}
dates: {lecDate}
lecturer: mmdarden
homeworkRelease: {hwDate}
solutionRelease: {solDate}
# ohppRelease: {lecDate}
title: "{title}"
# lectureTopics:
#   - TBD
# homeworkTopics:
#   - TBD
# advancedTopics:
#   - TBD
lectureSummary:
leccapWed:
leccapFri:
---
"""
```

That way OH++ notes don't get shown unless someone explicitly uncomments them. Or maybe it'd be better to break the build unless explicitly ignored. @the-alex thoughts?